src/CMakeLists.txt

```diff
@@ -323,6 +321,21 @@ if(Qt5Gamepad_FOUND)
   message(STATUS "Using Gamepad module")
 endif(Qt5Gamepad_FOUND)
 
+if(USE_OWN_QT5KEYCHAIN)
+  # An extra definition is needed if we are using our own QtKeyChain
+  target_compile_definitions(mudlet PRIVATE INCLUDE_OWN_QT5_KEYCHAIN QTKEYCHAIN_NO_EXPORT)
```

Comment:
I'd move this to the other `target_compile_definitions` above.

Reply:
Those above ones are unconditional whereas ~this one is~ both of them are dependent on being used for only for the built in QtKeychain - indeed the first one is what controls this feature in the compilable code.